FirmataParser.cpp

```diff
@@ -394,14 +413,24 @@ void FirmataParser::processSysexMessage(void)
 {
   switch (dataBuffer[0]) { //first byte in buffer is command
     case REPORT_FIRMWARE:
-      if (currentReportFirmwareCallback)
-        (*currentReportFirmwareCallback)(currentReportFirmwareCallbackContext);
+      if (currentReportFirmwareCallback) {
+        size_t sv_major = dataBuffer[1], sv_minor = dataBuffer[2];
```

Comment:
No. I would have compiled in the constants, they describe the version on my side of the protocol not the other side.